stockmetrics/metricsDescriptions/metricsDescriptions/BlankLinesCountClass.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+<table><tr><td valign="top" height="150">
+    <font face="verdana" size="-1">
+        The number of blank lines in class's code.
```

Comment:
Seems like all such descriptions start with "Calculates...". I think we should stick to that.